Main.c

```diff
@@ -0,0 +1,72 @@
+//Attachments area
+#include<stdio.h>
+#include<string.h>
+#include<stdlib.h>
+//print Header information
+void printHeader(int buffer)
+{
```

Comment:
I wanted all the printing logic inside this API. Call print header which will parse the 4 bytes of data and print.